services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -514,6 +515,7 @@ impl NetCdfCfDataProvider {
                 end, // TODO: Use this or time dimension size (number of steps)?
                 step,
                 band_offset: dataset_id.entity * dimensions_time,
+                cache_ttl: CacheTtl::default(),
```

Comment:
could also be configurable

Reply:
5c04d280f009ec129d8b8fb5a92e3fc1ca61c65b